6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,20 @@
+import { createRequire } from "module";
+const require = createRequire(import.meta.url);
```

Comment:
These imports are not needed, as you do not need to use any libraries for the exercise. Removing these lines your code runs perfectly. 😄